File: Assignment24/Program4.c

//problem Statement :Accept character from user and check wheather it is alphabet or not (A-Z,a-z)

#include<stdio.h>
#define TRUE 1
#define FALSE 0

typedef int BOOL;

//Input  : g
//Output : TRUE

//Input : D 
//Output : FALSE

BOOL SmallCase(char ch)
{
   if(ch >='a' && ch<='z')
   {
        return TRUE;
   }
   else
   {
       return FALSE;
   }
}


int main()
{
  char cValue = '\0';
  BOOL bRet = FALSE;
  
  printf("Enter the character:");
  scanf("%c" ,&cValue);

  bRet  = SmallCase(cValue); //Function call
  if(bRet == TRUE)
  {

      printf("It is Small case character");

  }
  else
  {
      printf("It is not Small case character");
  }

    return 0;
}

File: Assignment32/Program4.java


import java.util.*;


//Problem statement

//Write a java program Which accept string from user and check wheather it conatins vowels in it or not




class Program4
{
    
    public static boolean ChkVowels (String str)
    {
           int  icnt = 0;
           int  iCount = 0;
           char Arr[] = str.toCharArray();
           
           for(icnt =0; icnt < Arr.length; icnt++)
           {
               if(Arr[icnt] == 'a' || Arr[icnt] == 'e' ||  Arr[icnt] == 'i' || Arr[icnt] == 'o' || Arr[icnt] == 'u' || Arr[icnt] == 'w')
               {
                   return true;
               }
                
           }
           return false;
    }

    public static void main(String Arg[])
    {
      Scanner sobj = new Scanner(System.in);

      String str = null;
      boolean bRet = false;
       
     try{
       
      System.out.println("Enter String :");
      str = sobj.nextLine();

       
      bRet = ChkVowels(str);
      if(bRet == true)
      {
        System.out.println("true");
      }
      else
      {
          System.out.println("false");
      }
      
    }
       catch(ArrayIndexOutOfBoundsException obj)
       {
        System.out.println("Array index out of bounds exception");

        
       }

       sobj.close();
      
    }
   
}


